### ISSUE TITLE
Notebook brightness hotkeys doesn't work

### ISSUE THREAD

**Issue by Atalanttore** [lxqt/lxqt#770]
Hi,

I want to change the brightness of my screen, but the dedicated notebook hotkeys doesn't work in LXQt though it works with KDE.

Regards,
Ettore

**Comment by stefonarch** [comment]
You can try in global shortcuts -> new pressing the buttons of your
notebook - it should recognise them as xf86MonBrightnessDown/Up - and
insert for decrease/increase
xbacklight -20
xbacklight +20

Obviously you need xbacklight installed, depends on your distro.



Il 03/08/2015 13:55, Atalanttore ha scritto:

**Comment by Atalanttore** [comment]
I've installed xbacklight and added both xf86MonBrightnessDown/Up to global shortcuts.

It works now, but why aren't the brightness buttons configured by default?

**Comment by jleclanche** [comment]
Some systems get brightness handling at the kernel/systemd level.

**Comment by Vladimir-csp** [comment]
And some systems handling it on hardware level. And behavior can be tweaked via kernel cli arguments. In this case xbacklight will not interfere.
But is there a way to check if kernel/systemd is in charge of brightness?

**Comment by jleclanche** [comment]
That's what I'd like to know.

**Comment by pmattern** [comment]
Basically this is a duplicate of #36, much as #504 which could be considered a spin-off regarding keyboard.
Just saying so as to avoid discussions in parallel.

**Comment by Allanitomwesh** [comment]
It would be really great if the keyboard hotkeys worked by default on LXQt. This is currently the most standout "missing" feature, along with only having volume slider in the toolbar panel but no brightness slider. 
I think brightness up down/ volume up/down and mute buttons working by default would be enough to give anyone in need of a lightweight DE peace.

**Comment by isf63** [comment]
Outdated, brightness and backlight and their hotkeys have been functional for some time. The current unimplemented spots are DDC/CI with external monitors (unrelated to laptops), and setting "brightness" (pixel color brightness) in Wayland.
